mobify/bellows
# Apply Modern Standards

Reviewers: @jansepar @fractaltheory @mikenikles @MikeKlemarewski 

## Changes
Update Bellows to some modern best-practices
- Prep for publishing to NPM
- Remove Zepto support in favor of jQuery
- Migrate all dependencies to NPM, remove Bower
- Migrate JS linting to eslint
- Migrate SCSS compilation to libsass
- Lint SCSS with sass-lint
- Update circleCI config with modern testing steps
- Add some GitHub templates for contributing, issues, and PRs

## Todos:
- [x] Add a PR template
- [x] +1 (Major version bump)
- [x] Update `package.json` and Readme with a non-conflicting NPM package name

## How to Test
- Checkout code
- Refresh node modules with `rm -rf node_modules && npm install`
- Test with `grunt test` or `grunt test:browser`
- Check that the examples still work with `grunt examples`
- Check out [this branch of Extra Stores](https://github.com/mobify/extra-mobile/tree/DNM-testing-modern-bellows) that uses this new code for its Bellows
